_projects/armv7a7-imx6ull-evk/rootfs-overlay/etc/rc.psh

```diff
@@ -4,6 +4,8 @@ export HOME=/root
 W /bin/bind devfs /dev
 X /sbin/imx6ull-gpio
 W /sbin/dummyfs -m /var -D
+// suppressing dmesg message from lwip before execution
```

Comment:
Use # for commented lines in psh scripts ([see](https://github.com/phoenix-rtos/phoenix-rtos-utils/blob/7b6b834b43cc1a5df57c10b5f9756cb973a9ba25/psh/pshapp/pshapp.c#L1115-L1116)).
```suggestion
# suppressing dmesg message from lwip before execution
```